spacy/pipeline/entity_linker.py

```diff
@@ -56,6 +56,7 @@
         "overwrite": True,
         "scorer": {"@scorers": "spacy.entity_linker_scorer.v1"},
         "use_gold_ents": True,
+        "abstention_threshold": None,
```

Comment:
Is there a difference in behavior between `None` and 0 here? If not, shouldn't it just default to 0?

Reply:
It's possible for the scores to be below zero. It's likely practically not relevant, but can happen (e.g. before running a single `update()`). In that case these scores would be discarded, even though no `threshold` might have been set. I think it'd be more reasonable to expect that no threshold check is run if the user doesn't set it, hence `None` instead of `0`.

Reply:
OK, makes sense. It would be good to mention that somewhere in a comment or docs.

Reply:
It's mentioned in the docstrings, will also write about it in the docs.